setup.sh

```diff
@@ -34,20 +34,7 @@ the version / tag of docker-mailserver. Please read the
 ly and use ./setup.sh help and read the VERSION section.\n" >&2
 }
 
-function _get_absolute_script_directory
-{
-  if dirname "$(readlink -f "${0}")" &>/dev/null
-  then
-    DIR="$(dirname "$(readlink -f "${0}")")"
-  elif realpath -e -L "${0}" &>/dev/null
-  then
-    DIR="$(realpath -e -L "${0}")"
-    DIR="${DIR%/setup.sh}"
-  fi
-}
-
-DIR="$(pwd)"
-_get_absolute_script_directory
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
```

Comment:
Any reason not to use `dirname “${BASH_SOURCE[0]}“` directly?

Reply:
I'll add back the old code, good catch 👍